src/planscape/conditions/views.py

```diff
@@ -1,6 +1,9 @@
-from django.http import HttpRequest, HttpResponse, QueryDict
+from django.http import HttpRequest, HttpResponse, JsonResponse, QueryDict
 from django.db import connection
 
+import json, os
+
+PLANSCAPE_ROOT_DIRECTORY = '/Users/jessxlee/cnra/'
 
 # Name of the table and column from models.py.
 RASTER_TABLE = 'condition_raster'
```

Comment:
you'll want to merge from main before submitting!

Reply:
Done!